app/src/views/layouts/full-width-landmarks.njk

```diff
@@ -0,0 +1,10 @@
+{% extends "layout.njk" %}
+
+{% block main %}
+  <div class="govuk-width-container">
+    {% block beforeContent %}{% endblock %}
+  </div>
+  <main class="govuk-main-wrapper {{ mainClasses }}" id="main-content" role="main">
```

Comment:
I've created `full-width.njk` (without wrapping landmarks) alongside `full-width-landmarks.njk`

This is so Axe can pass some of the "landmark-in-landmark" violations

You'll see some Percy related diff changes as a result